### ISSUE TITLE
Drag and drop over a PDF object is currently broken in Chrome for Mac

### ISSUE THREAD

**Issue by sballesteros** [PREreview/rapid-prereview#13]
See https://bugs.chromium.org/p/chromium/issues/detail?id=984891&q=drag%20object&colspec=ID%20Pri%20M%20Stars%20ReleaseBlock%20Component%20Status%20Owner%20Summary%20OS%20Modified
Issue is active and should be fixed promptly per the comments. 

If not, a possible workaround is to resize the `<object>` so that the PDF is not below the shell when the user starts to drag and to resize it back to 100vh after the drop happened.

**Comment by sajacy** [comment]
May I suggest that the drag-and-drop interaction be removed completely?  

IMHO, the accessibility and UX would be improved by simplifying to a circular highlight border around the profile picture to indicate if the review is shown. To preserve the full functionality in the menu, each profile photo could also have a "External Link" icon (&#x2B08;) near the photo that links to the reviewer's profile (improved with `target="_blank"`).

**Comment by georgiamoon** [comment]
I 100% agree! That sounds great.

**Comment by dasaderi** [comment]
I also agree. Even though it was a clever idea, it's been a feature no user has got in our interviews and I myself struggle with it.
